canonical/maas-ui
# chore: Refactor Tags to use PageContent MAASENG-1829

## Done

- Refactored Tags to use PageContent

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to Tags
- Click "Create new tag"
- Ensure form opens correctly
- Click on a tag
- Click "Delete"
- Ensure form opens correctly

## Fixes

Fixes [MAASENG-1829](https://warthogs.atlassian.net/browse/MAASENG-1829)


[MAASENG-1829]: https://warthogs.atlassian.net/browse/MAASENG-1829?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
